biorbd_optim/gui/plot.py

```diff
@@ -423,13 +423,13 @@ def __update_axes(self):
                     if not self.axes[key][0].ylim:
                         y_max = -np.inf
                         y_min = np.inf
-                        for p in ax.get_children():
-                            if isinstance(p, lines.Line2D):
-                                y_min = min(y_min, np.min(p.get_ydata()))
-                                y_max = max(y_max, np.max(p.get_ydata()))
-                            y_range, data_range = self.__compute_ylim(y_min, y_max, 1.25)
-                            ax.set_ylim(y_range)
-                            ax.set_yticks(np.arange(y_range[0], y_range[1], step=data_range / 4,))
+                        children_list = [p for p in ax.get_children() if isinstance(p, lines.Line2D)]
+                        for p in children_list[:-2]:
```

Comment:
Can you add a comment on why the last 2 should not be tested? I don't think this is self-explanatory